components/transaction/zksync/era/EraTransferLineItem.vue

```diff
@@ -2,7 +2,7 @@
   <TransactionLineItem :icon="icon" :explorer-url="blockExplorerUrl" :transaction-hash="transfer.transactionHash!">
     <template #top-left>{{ label }}</template>
     <template #bottom-left>
-      <span>{{ time }}</span>
+      <span data-testid="withdraw-date">{{ time }}</span>
```

Comment:
I remember that such kind of changes requires Danil's review. 
I will ping him for it after QA part review - in order to avoid re-approves from him after some changes will be made from testing side.